test/loopback-dl.test.js

```diff
@@ -1473,6 +1483,20 @@ describe('DataAccessObject', function() {
     assert(error, 'An error should have been thrown');
   });
 
+  it('should throw when malformed logical operators follow valid logical clauses', function() {
```

Comment:
Hey @Amir-61, I'm happy to make this change, however - this seems inconsistent with the grammar of the rest of the tests which all include the 'should' prefix. What do you advise?